particles.py

```diff
@@ -0,0 +1,149 @@
+"""
+particles.py
+A python package for the MolSSI Software Summer School.
+Contains Particles class and subclasses RandomParticles and FetchParticles
+"""
+
+import numpy as np
+from box import *
+from abc import ABC, abstractmethod
+
+class Particles(ABC):
+    ''' Abstract Particles class
+    '''
+
+    def __init__(self):
+        '''The constructor of the Particles Class.
+        '''
+        self._num_particles = 0
+        self._coordinates = []
+
+    def __str__(self):
+        return( F'Particles Object: {self._num_particles} particles.' )
+
+    @abstractmethod
+    def _generate_coordinates(self):
+        '''Generates the coordinates of the Particles Object '''
+        pass
+
+    @property
+    def coordinates(self):
```

Comment:
Making `coordinates` a property doesn't give us anything that I can tell. We aren't updating anything with coordinates,
and we do need to directly update a given particle's coordinates.